.github/workflows/build_test_deploy.yml

```
@@ -1,13 +1,208 @@
+        run: |
+          for i in {1..20}; do # retry
+            curl --fail -v http://localhost:9050/tracings/health && s=0 && break || s=$?
+            sleep 5
+          done
+          (exit $s)
+
+      - name: Stop webknossos, datastore, and tracingstore
+        run: docker compose down --volumes --remove-orphans
+
+      - name: Push docker images
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2034:warning:2:3: i appears unused. Verify use (or export if used externally)

(shellcheck)